### PR TITLE
feat: store previously-visited custom networks

### PR DESCRIPTION
**Pull request by mvadari**
## High Level Overview of Change

Title says it all.

### Context of Change

You don't need to keep entering the same URLs over and over again, there's a handy link.

### Type of Change

- [x] New feature (non-breaking change which adds functionality)

### TypeScript/Hooks Update

N/A

### Custom Network Page
Before:
image

After:
image

### Network Picker
Before:
image

After:
image

## Test Plan

Works locally.

### CODE REVIEW

File: src/containers/Header/NetworkPicker/NetworkPicker.tsx [hunk 1]
```diff
@@ -116,12 +140,12 @@ export const NetworkPicker = () => {
       <>
         {networks.map(({ network, title, url = '' }) => {
           if (
-            network === currentMode ||
+            (currentMode !== 'custom' && network === currentMode) ||
```
Comment (ckniffen) [inline src/containers/Header/NetworkPicker/NetworkPicker.tsx:143]: Can you add a comment explaining what is going on here.  I forgot what the reasoning is.

File: src/containers/Header/NetworkPicker/NetworkPicker.tsx [hunk 2]
```diff
@@ -69,6 +76,23 @@ export const NetworkPicker = () => {
         handler={handleNetworkClick(network)}
       >
         {text}
+        {network === 'custom' && (
```
Comment (ckniffen) [inline src/containers/Header/NetworkPicker/NetworkPicker.tsx:79]: Use `isCustom` instead.

**Comment by ckniffen** [comment]
Should you be able to remove the custom networks on the homepage?

**Review by ckniffen — APPROVED** [review]

**Comment by mvadari** [comment]
> Should you be able to remove the custom networks on the homepage?

That wasn't part of the initial design, but I could add that if you want (replace the arrows with an X)

**Comment by ckniffen** [comment]
> > Should you be able to remove the custom networks on the homepage?
> 
> That wasn't part of the initial design, but I could add that if you want (replace the arrows with an X)
I think it is fine for now just something to consider.

**Review by pdp2121 — APPROVED** [review]

**Pull request merged**